app/models/species.rb

```diff
@@ -1,4 +1,4 @@
-class Specie < ApplicationRecord
+class Species < ApplicationRecord
```

Comment:
The word 'species' appears to be an irregular noun so it's both plural and singular at the same time, how  are words like these handled in Rails? Should we leave the model as 'Species' or use an inflection like @joseantoniordlmc suggested?